llm_bench/python/utils/model_utils.py

```diff
@@ -30,14 +31,71 @@ def get_prompts(args):
                     with open(input_prompt, 'r', encoding='utf-8') as f:
                         for line in f:
                             data = json.loads(line)
-                            prompts_list.append(data['prompt'])
+                            if 'prompt' in data:
+                                if data['prompt'] != '':
+                                    prompts_list.append(data['prompt'])
+                                else:
+                                    raise RuntimeError('== prompt should not be empty string ==')
+                            else:
+                                raise RuntimeError('== key word "prompt" does not exist in prompt file ==')
                 else:
                     raise RuntimeError('== The prompt file does not exist ==')
             else:
-                prompts_list.append(input_prompt)
+                raise RuntimeError('== The prompt file should be ended with .jsonl ==')
     return prompts_list
 
 
+def init_image_param():
+    return {'prompt': '', 'width': 0, 'height': 0, 'steps': 0, 'guidance_scale': ''}
+
+
+def get_image_param_from_prompt_file(args):
+    image_param_list = []
+    image_param = init_image_param()
```

Comment:
How about we don't initialize them here, since the parameter will be missed if it is not in prompt file.  In https://github.com/openvinotoolkit/openvino.genai/pull/66/files#diff-e4414c499d572139aee57a51241ee880d46d88c03137a1142d534ebf8494e5f1R315-R322
resize_image_width = img.get('width', DEFAULT_SUPER_RESOLUTION_WIDTH)
[https://docs.python.org/3/library/stdtypes.html#dict  get(key[, default])[(https://docs.python.org/3/library/stdtypes.html#dict.get)](https://docs.python.org/3/library/stdtypes.html#dict.get)

Reply:
Done